Fatal error: Uncaught exception 'Exception' with message 'Return exception handle' in %s:6
Stack trace:
#0 %s(14): Hack\UserDocumentation\Async\Exceptions\Examples\BasicException\exception_thrower()
#1 %s(17): Hack\UserDocumentation\Async\Exceptions\Examples\BasicException\basic_exception()
#2 {main}
